radicle-cli/examples/workflow/5-patching.md

```diff
@@ -0,0 +1,69 @@
+Back to being the project maintainer.
+
+Changes have been proposed by another person (or peer) via a radicle patch.  To follow changes by another, we must 'track' them.
+
+```
+$ rad track did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk --alias bob
+✓ Tracking policy updated for z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk (bob)
+! Warning: fetch after track is not yet supported
+$ rad fetch
+✓ Fetching rad:z42hL2jL4XNk6K8oHQaSWfMgCL7ji from z6Mkt67…v4N1tRk..
+✓ Fetched repository from 1 seed(s)
+```
+
+Additionally, git must know about the new peer.  We need to add a new 'git
+remote' for this.  Upcoming versions of radicle will not require this step.
+
+```
+$ git remote add bob rad://z42hL2jL4XNk6K8oHQaSWfMgCL7ji/z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk
+$ git fetch bob
+```
+
+The contributors changes are now visible to use.
+
+```
+$ git branch -r
+  bob/flux-capacitor-power
+  bob/master
+  rad/master
+$ rad patch
+
+❲YOU PROPOSED❳
+
+Nothing to show.
+
+❲OTHERS PROPOSED❳
+
+Define power requirements a07ef7743a3 R1 27857ec ahead 2, behind 0
+└─ * opened by did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk 2 months ago
+└─ * patch id a07ef7743a32a2e902672ea3526d1db6ee08108a
+
+```
+
+Wait! There's a mistake.  The REQUIREMENTS should be markdown file.  Lets
+quickly update the patch before incorporating the changes.  Updating it this
+way will tell others about the corrections we needed before merging the
+changes.
+
+```
+$ git checkout flux-capacitor-power
+branch 'flux-capacitor-power' set up to track 'bob/flux-capacitor-power'.
+$ git mv REQUIREMENTS REQUIREMENTS.md
+$ git commit --fixup HEAD
+[flux-capacitor-power f5648be] fixup! Add README, just for the fun
+ 1 file changed, 0 insertions(+), 0 deletions(-)
+ rename REQUIREMENTS => REQUIREMENTS.md (100%)
+$ rad patch update --message "Define power requirements" --message "See details." a07ef7743a32a2e902672ea3526d1db6ee08108a
+
+🌱 Updating patch for heartwood
+
+✓ Pushing HEAD to storage...
+✓ Analyzing remotes...
+
+a07ef7743a3 R1 (27857ec) -> R2 (f5648be)
+1 commit(s) ahead, 0 commit(s) behind
+
+
+✓ Patch a07ef7743a32a2e902672ea3526d1db6ee08108a updated 🌱
+
+```
```

Comment:
Shouldn't we merge and close the issue as the last step?

Reply:
Its planned to be part of finishing `rad merge` from #148

Reply:
These changes are a part of supporting `rad update` by others.  I have changes prepared which include a third collaborator, something more natural, but it appears to have surfaced a bug.